libnmstate/state.py

```diff
@@ -554,6 +561,18 @@ def _validate_routes(
                 )
 
 
+def compress_ipv6_addr(ipv6_address_prefix_len):
```

Comment:
This function does not actually need the `ADDRESS_PREFIX_LENGTH`.
You may simplify it to just take the ip address string as input parameter.

Reply:
I think that @maiqueb wanted to convert the address dict as a whole in order to simplify the usage:
`ifstate['ipv6'][InterfaceIPv6.ADDRESS] = [compress_ipv6_addr(ipv6_addr) for ipv6_addr in ipv6_addrs]`

I guess if he will use a for-loop at the caller, he can avoid this.
I am ok with both, it reads well IMO.

Reply:
I think the caller reads better this way.